build/azure-pipelines/common/publish.ts

```diff
@@ -293,15 +292,18 @@ function main(): void {
 		return;
 	}
 
+	const commit = process.env['BUILD_SOURCEVERSION'];
+
+	if (!commit) {
+		console.warn('Skipping publish due to missing BUILD_SOURCEVERSION');
+		return;
+	}
+
 	const opts = minimist<PublishOptions>(process.argv.slice(2), {
 		boolean: ['upload-only']
 	});
 
-	// {{SQL CARBON EDIT}}
```

Comment:
@MattIrv I wasn't sure what this edit was for, can you confirm this is still correct?

Reply:
we'll need to update our release scripts to set `BUILD_SOURCEVERSION` instead of calling the scripts with a commit argument but it looks fine as long as we do that

Reply:
I'll have to check but I'm pretty sure that env is set by default in azure pipelines

Reply:
Oh right, just confirmed that it gets set in our releases. This should be fine then